assign.py

```diff
@@ -227,21 +236,21 @@ def main():
         if options.procstring:
             procstring = options.procstring
         elif not taskchain:
-            procstring = wf.info['ProcessingString']
+            procstring = wf_info['ProcessingString']
         if options.era:
             era = options.era
         elif not taskchain:
-            era = wf.info['AcquisitionEra']
+            era = wf_info['AcquisitionEra']
         #Set era and procstring to none for merge ACDCs inside a task chain
-        if schema["RequestType"] == "Resubmission" and wf.info["PrepID"].startswith("task") and "Merge" in schema["InitialTaskPath"].split("/")[-1]:
+        if schema["RequestType"] == "Resubmission" and wf_info["PrepID"].startswith("task") and "Merge" in schema["InitialTaskPath"].split("/")[-1]:
             era = None
             procstring = None
 
         # Must use --lfn option, otherwise workflow won't be assigned
         if options.lfn:
             lfn = options.lfn
-        elif "MergedLFNBase" in wf.info:
-            lfn = wf.info['MergedLFNBase']
+        elif "MergedLFNBase" in wf_info:
+            lfn = wf_info['MergedLFNBase']
```

Comment:
lfn needs to be taken from the original request. This line is reading the MergedLFNBase value from the request to be assigned, makes no sense.